edb/pgsql/common.py

```diff
@@ -122,7 +122,9 @@ def quote_type(type_):
 
 
 def get_module_backend_name(module, prefix='edgedb_'):
-    return edgedb_name_to_pg_name(f'{prefix}{module}', len(prefix))
+    # We now put everything in the same schema, but haven't yet cleaned up
+    # all the call sites.
+    return "edgedb"
```

Comment:
We should pick a different schema name.  `edgedb` is reserved for internal structures, functions etc.  Additionally, the standard library should also use a separate schema.  The reason here is simple: dumps, logical replication, upgrades etc are all much simpler if we cleanly separate system structures from user-defined structures.

I propose:

`edgedbstd`: standard library and internal helpers (`metaschema`)
`edgedbinstdata`: unique instance data (already present)
`edgedbpub`: user data

Reply:
+1. Except I'd change `edgedbinstdata` to edgedbinst` (or `edgedbpub` to `edgedbpubdata`). I like the former.

Reply:
OK, I did a split between `edgedbstd` for the standard library (everything in STD_MODULES) and `edgedbpub` for everything else.

I haven't moved the metaschema helpers from `edgedb` to `edgedbstd`; that seems orthogonal to this change.